Libraries/Text/TextInput/Multiline/RCTUITextView.m

```diff
@@ -27,7 +27,11 @@ @implementation RCTUITextView {
 
 static UIFont *defaultPlaceholderFont()
 {
+#if !TARGET_OS_OSX // [macOS]
   return [UIFont systemFontOfSize:17];
+#else // [macOS
+  return [UIFont systemFontOfSize:[NSFont systemFontSize]];
```

Comment:
```suggestion
  return [NSFont systemFontOfSize:[NSFont systemFontSize]];
```

Reply:
how does that work? do we macro change uifont to nsfont in macOS? we should change the function signature as well?

Reply:
Just updated to ifdef the function declaration as well. @Saadnajmi just curious, I tried to find in the rnmacOS code where the macro for this was but couldn't find it. Do you know where it is?

Reply:
Should be in RCTUIKit.h

Reply:
Ah I see, never seen `@compatibility_alias` before

Reply:
https://gcc.gnu.org/onlinedocs/gcc-4.5.4/gcc/compatibility_005falias.html

It is... very sparsely documented :P

Reply:
Hence I'm in favor of getting rid of it and replacing it with a proper `RCTUIFont` shim like we've been doing elsewhere.